content/applications/hr/payroll/reporting.rst

```diff
@@ -2,218 +2,309 @@
-:guilabel:`DMFA` reports are universal and available for all companies regardless of location. The
-:guilabel:`Paid Time Off Allocation`, :guilabel:`273S Sheet`, and :guilabel:`274.XX Sheets` reports
-are specific to Belgian companies only.
+The :guilabel:`Reporting` section of the *Payroll* app offers a variety of reports to choose from,
+organized by location.
 
-To view a report, go to :menuselection:`Payroll --> Reporting` and click on the specific report.
+The *Payroll* report, *Work Entry Analysis* report, and *Salary Attachment Report* are default
+reports in the *Payroll* app, and are available for all companies, regardless of location.
 
-.. image:: reporting/reports.png
-   :align: center
-   :alt: Report dashboard view.
-
-If a report is unavailable to a company, a user error pops up, stating `You must be logged in to a
-Belgian company to use this feature.`
+Beneath the three default reports are all localization-based reports, organized by country, in
+alphabetical order. These reports contain all the various information for the offered benefits and
+local tax laws.
 
-At the top of each report, click on :guilabel:`Filters` to display the optional filters for the
-specific report. Filters show information that matches the specific filter parameters. For example,
-when selecting the :guilabel:`Last 365 Day Payslip` filter, only payslips for the last 365 days
-appear. All reports have the option to add a custom filter, and each have different default custom
-filter options. Select the parameters, then click the :guilabel:`Apply` button.
+To view all the available reports for the database, including all the localization-specific ones,
+navigate to :menuselection:`Payroll app --> Reporting` to view the available reports in a drop-down
+menu. Click on a specific report to view it.
 
-.. image:: reporting/custom-filter.png
+.. image:: reporting/reports.png
    :align: center
-   :alt: Report dashboard view.
+   :alt: Report dashboard view showing extra reports for Belgium databases.
 
-Clicking on :guilabel:`Favorites` displays three options: :guilabel:`Save the current search`,
-:guilabel:`Add the search to the dashboard`, or :guilabel:`Insert the search to a Google
-spreadsheet`.
+If a report is unavailable to a user, an :guilabel:`Invalid Operation` pop-up window appears,
+stating: :guilabel:`You must be logged in to a (country) company to use this feature.`
 
-.. image:: reporting/favorites.png
-   :align: center
-   :alt: Favorite options for reports.
+Default reports
+===============
 
 Payroll
 -------
 
-Click on :menuselection:`Payroll --> Reporting --> Payroll` to display the :guilabel:`Payroll
-Analysis` report.
+Click on :menuselection:`Payroll app --> Reporting --> Payroll` to display the :guilabel:`Payroll
+Analysis` report. This report shows all the payslips generated in the last 365 days, due to the
+default :ref:`filter <payroll/filters>`: :guilabel:`Last 365 Days Payslip`.
```

Comment:
But does that apply when it is on-screen as the default? This is what they see in that field when they first go to that report, so they are not typing it in. And since it is the default- it also appears as a 'button' in the drop-down, if you remove the filter:
![image](https://github.com/odoo/documentation/assets/121518652/0070b048-d87c-44fa-9f1b-14932f9847fd)

Reply:
I removed it, but I just wanted to explain why I made it a gui - only because it appears in the box as something you can click off:
![image](https://github.com/odoo/documentation/assets/121518652/998df39d-3775-4316-ba96-9c8def16294b)